File: Asocioacion/python/cuartoEjercicio/Automotor.py
class Automotor:
    
    c = 0
    
    def __init__(self, marca, modelo, precio):
        self.marca = marca
        self.modelo = modelo
        self.precio = precio
        self.__class__.c+=1
     
    #Marca del automotor 
    def getMarca(self):
        return self.marca
    
    def setMarca(self, marca):
        self.marca = marca
        
    #Modelo del automotor
    def getModelo(self):
        return self.modelo
    
    def setModelo(self, modelo):
        self.modelo = modelo
        
    #Precio del automotor
    def getPrecio(self):
        return self.precio
    
    def setPrecio(self, precio):
        self.precio = precio
        
    #Metodo de saludar
    def saludarAutomotor(self):
        print('AUTOMOTORES')
        print('-'*50)
        print(f'La marca del automotor es: {self.marca}')
        print(f'El modelo del automotor es: {self.modelo}')
        print(f'El precio del automotor es: {self.precio}')                        

File: Asocioacion/php/cuartoEjercicio/Automotor.php
<?php

class Automotor{
    
    private string $marca;
    private int $modelo;
    private float $precio;
    
    public function __construct(string $marca, int $modelo, float $precio){

        $this->marca = $marca;
        $this->modelo = $modelo;
        $this->precio = $precio;

    }

    //Marca del automotor
    public function getMarca(){
        return $this->marca;
    }
    public function setMarca($marca){
        $this->marca = $marca;
    }

    //Modelo del automotor
    public function getModelo(){
        return $this->modelo;
    }
    public function setModelo($modelo){
        $this->modelo = $modelo;
    }

    //Precio del automotor
    public function getPrecio(){
        return $this->precio;
    }
    public function setPrecio($precio){
        $this->precio = $precio;
    }

    //Metodo de mostrar datos

    public function mostrarDatos(){
        echo '<h2>'.'<b>'.'AUTOMOTORES'.'</b>'.'</h2>';        
        echo 'La marca del automotor es: '.$this->marca.'<br>';
        echo 'El modelo del automotor es: '.$this->modelo.'<br>';
        echo 'El precio del automotor es: '.$this->precio.'<br>';
    }    

}

?>